### ISSUE TITLE
Improvements

### ISSUE THREAD

**Issue by tomastraini** [AlexPetrusca/Mephisto#20]
I took it for myself to make some improvements to mephisto for Windows 10 (chrome, chess.com)

https://github.com/tomastraini/Mephisto-master

The improvements i did are:
- Update Stockfish from the weaker web stockfish to desktop v15.1 via Python API
- Add a book moves system
- New option to search by movetime or depth
- Solved promotion bug (table not updating after a promotion)
I guess that other improvement would be to make the python API's stockfish usable in Linux and not simply a .exe

**Comment by toichimto** [comment]
Thank you for the update, idk why but the extension keep saying "calculating" and it is not running properly, can you help me with this issue?

**Comment by tomastraini** [comment]
Yes! In order to run it, you need to have python 3.10 installed in your PC. (https://www.python.org/downloads/)
Once installed and with python in your PATH (environment variable, the installer lets you do it automatically on windows), and run the stockfishapy.py file, inside the Stockfish API folder. It might tell you at first that some files or libraries were not found.
For that you have to run "pip install (the package it tells you it needs)
For example one of the needed packages is request, so you would run "pip install request", and it would do it for you.

**Comment by tomastraini** [comment]
These are probably the packages needed to install with pip:
- flask
- flask_cors
- requests
- chess
Ps: i have a version to run it on linux but i still have to push it, sooner than later i'll send one that also works on Ubuntu

**Comment by IchNukeDichWeg** [comment]
> > Yes! In order to run it on Windows, you need to have python 3.10 installed in your PC. (https://www.python.org/downloads/) Once installed and with python in your PATH (environment variable, the installer lets you do it automatically on windows), go to the Stockfish API/ folder inside the Mefisto-main folder, and run the stockfishapi.py file, (open a CMD and run "py stockfishapi.py". It might tell you at first that some files or libraries were not found. For that you have to run "pip install (the package it tells you it needs)" For example one of the needed packages is request, so you would run "pip install request", and it would do it for you.
> 
> Do I have to run "py stockfishapi.py" in the Command Prompt or simply run the file in the download folder? Can you make an instruction video?

@MC-PO @tomastraini you can check my fork or PR it explains everything and is easy to use and has the newest engines
